packages/button/src/LionButton.js

```diff
@@ -140,32 +145,42 @@ export class LionButton extends DelegateMixin(SlotMixin(LionLitElement)) {
 
   connectedCallback() {
     super.connectedCallback();
-    this.__setupKeydownDelegation();
+    this.__setupDelegation();
   }
 
   disconnectedCallback() {
     super.disconnectedCallback();
-    this.__teardownKeydownDelegation();
+    this.__teardownDelegation();
   }
 
   __clickDelegationHandler(e) {
     e.stopPropagation(); // prevent click on the fake element and cause click on the native button
     this.$$slot('_button').click();
   }
 
-  __setupKeydownDelegation() {
+  __setupDelegation() {
     this.addEventListener('keydown', this.__keydownDelegationHandler);
+    this.addEventListener('keyup', this.__keyupDelegationHandler);
   }
 
-  __teardownKeydownDelegation() {
+  __teardownDelegation() {
     this.removeEventListener('keydown', this.__keydownDelegationHandler);
+    this.removeEventListener('keyup', this.__keyupDelegationHandler);
   }
 
   __keydownDelegationHandler(e) {
+    if (e.keyCode === 32 /* space */ || e.keyCode === 13 /* enter */) {
+      e.preventDefault();
+      this.shadowRoot.querySelector('.btn').classList.add('active');
```

Comment:
I could of course cache `this.shadowRoot.querySelector('.btn')`, but decided not to since this is a super minor pre optimisation for this particular case and will only make code more complex.

Reply:
Shouldn't it be a modifier (attribute) instead of a regular class?

Reply:
This was impossible to customize active style before, still impossible now. We can investigate this, but it is not part of this bugfix MR.

Reply:
For next button update: from Subclasser perspective (who theoretically should be able to override css) would be better to refer to unique identifiers instead of classes